source/advanced-topics.adoc

```diff
@@ -526,15 +503,9 @@ of the source tarball because it removes the following line from the script:
 [[setup-ab]]
 ==== %setup -a and -b
 
-Options ``-a`` and ``-b`` expand specific sources, where ``-b`` (before)
-expands them prior entering the working directory, while ``-a`` (after) after
-entering. Their arguments are source numbers from the SPEC file preamble.
+Options ``-a`` and ``-b`` expand specific sources. ``-b`` expands specific sources before entering the working directory, while ``-a`` expands those sources after entering. Their arguments are source numbers from the spec file preamble.
```

Comment:
Can you keep the (before) and (after)? -b stands for "before" so it is easier to memorize.

Reply:
It is a nice way to memorize those options. I have made new commit with this addition.